src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2.java

```diff
@@ -9,10 +9,7 @@
 import org.broadinstitute.hellbender.cmdline.programgroups.ShortVariantDiscoveryProgramGroup;
```

Comment:
The new walker would get a Mutect2Engine that would have somatic/mitochondrial as a parameter so it could pass that to its SomaticGenotypingEngine in order to use a key that's different from TLOD for the score.  It should be easy enough to rename the tumor-sample arg, but let me know if it gets weird.

Reply:
My concern isn't the credit it's the branding.  It's easier to tell people that our best practices for mitochondria is simply Mutect2 than it is to say it's a wrapper for Mutect2.